.github/renovate.json

```diff
@@ -11,6 +11,10 @@
             "groupName": "github-actions",
             "matchManagers": ["github-actions"],
             "separateMajorMinor": "false"
+        },
+        {
+            "groupName": "requirements-cibw.txt",
+            "matchFileNames": [".ci/requirements-cibw.txt"]
```

Comment:
Are you sure that renovate won't handle this automatically? https://docs.renovatebot.com/modules/manager/pip_requirements/#file-matching

Reply:
I'm not sure. Shall we leave this file as is, and check when the next release comes out?

Reply:
I don't think we'd need to wait until the next release. I think if we merged the PR without this change, then we could see if #6604 started listing the file or not.

Reply:
Changes to this file dropped.

Reply:
Yep, now that this is merged, it's there!